recipes/abacus/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "abacus" %}
+{% set version = "3.2.4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/deepmodeling/abacus-develop/archive/v{{ version }}.tar.gz
+  sha256: 65a84a4a02a085c42f86d54b487bf3f1501a45f940370a8a7527c881145f78d6
+
+build:
+  script: |
+    export CMAKE_PREFIX_PATH=`python -c "import torch;print(torch.__path__[0])"`/share/cmake:$CMAKE_PREFIX_PATH
+    cmake -B conda_build ${CMAKE_ARGS} -DCMAKE_BUILD_TYPE=Release -DENABLE_DEEPKS=1 -DENABLE_LIBXC=1
+    cmake --build conda_build -j`nproc`
+    cmake --install conda_build
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make
+    - cmake
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+
+  host:
+    - {{ mpi }}
+    - openblas
+    - elpa * mpi_{{ mpi }}_*
+    - fftw * mpi_{{ mpi }}_*
+    - cereal
+    - pytorch
+    - libxc
+
+  run:
+    - {{ mpi }}
+    - scalapack
+    - libopenblas
+    - openblas
+    - libxc-c
+
+test:
+  commands:
+    - abacus  # [mpi == 'mpich']
+    - mpirun -n 2 abacus  # [mpi == 'mpich']
```

Comment:
@njzjz I wonder why skipping tests for OpenMPI here

Reply:
let's see if the new commit works

Reply:
> let's see if the new commit works

It does not... Let's just skip that.